rapids-cmake/cpm/fmt.cmake

```diff
@@ -50,6 +50,11 @@ Result Variables
 #]=======================================================================]
 function(rapids_cpm_fmt)
   list(APPEND CMAKE_MESSAGE_CONTEXT "rapids.cpm.fmt")
+  include("${rapids-cmake-dir}/cmake/detail/policy.cmake")
+  rapids_cmake_policy(DEPRECATED_IN 25.08
+                      REMOVED_IN 26.04
```

Comment:
26.04 is a very long time away. We've always done removal within a 6 month window. Can we make this 25.10 or 25.12?

Reply:
I was pushing this out because the last time we deprecated something major we were asked to bring it back. I'm fine shortening this one though, I know that at least nvbench has stopped using `rapids_cpm_fmt` so maybe other consumers can be quickly convinced to do so as well. I can move to 25.12.